app/src/marketplace/modules/dataUnion/services.js

```diff
@@ -247,11 +247,12 @@ export const createClient = (options: CreateClient = {}) => {
         },
         sidechain: {
             url: process.env.DATA_UNION_SIDECHAIN_PROVIDER,
-            chainId: process.env.DATA_UNION_SIDECHAIN_ID,
+            chainId: parseInt(process.env.DATA_UNION_SIDECHAIN_ID, 10),
         },
         mainnet: {
             url: process.env.WEB3_PUBLIC_HTTP_PROVIDER,
         },
+        streamrNodeAddress: getStreamrEngineAddresses()[0],
```

Comment:
Why the `[0]` indexing.. Or what is this for?

Reply:
StreamrClient config only accepts one node address :/

Reply:
What is it used for actually? It's not documented in the client

Reply:
Yeah, not documented yet. Usage is here: https://github.com/streamr-dev/streamr-client-javascript/blob/21d668d228c560a0569908c19329220f0235b2a7/src/dataunion/DataUnion.ts#L507.